Makefile

```diff
@@ -2,6 +2,7 @@ T=openssl
 
 PREFIX		?=/usr/local
 LIB_OPTION	?= -shared 
+CC			 = gcc
```

Comment:
If you edit the file right here, in GitHub, and set the tab size to 4, you will see that my lines are well aligned. So I can't do anything else in that sense. The problem is that you cannot always change the width of shown tabs.